README.md

```diff
@@ -68,15 +68,6 @@ For more information check the FIWARE Catalogue entry for [Robotics](https://git
 Write to evaluation.support@eprosima.com or mention @EProsima on Twitter.
 We are curious to get to know your use case!
 
-## Supported platforms
-
-More information about the official support can be found [here](https://github.com/eProsima/Fast-DDS/blob/master/PLATFORM_SUPPORT.md#platform-support)
```

Comment:
I don't think we should remove the link to the supported platforms, just the jenkins CI badges